src/stores/appStore.js

```diff
@@ -1,20 +1,35 @@
+import { useBase64, useColorMode, useLocalStorage } from '@vueuse/core';
```

Comment:
Esse método não parece estar sendo usado: `useBase64`

```suggestion
import { useColorMode, useLocalStorage } from '@vueuse/core';
```